low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -274,30 +275,30 @@ class SSPLDevDeploy:
         from cortx.utils.service import Service
         from cortx.utils.validator.v_service import ServiceV
         from cortx.utils.validator.error import VError
-        from cortx.sspl.lowlevel.framework import sspl_rabbitmq_reinit
         try:
             ServiceV().validate("isrunning", ["rabbitmq-server"])
         except VError:
             Service('dbus').process('start', 'rabbitmq-server.service')
-        sspl_rabbitmq_reinit.main(self.PRODUCT_VERSION)
         print("INFO: Done rabbitmq setup.")
 
     def setup_sspl(self):
         """Initialize SSPL service."""
-        from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_setup import main
+        setup_script = "%s/bin/sspl_setup" % (SSPL_BASE_DIR)
+
         commands = [
-            "sspl_setup post_install --config %s" % (self.sample_global_config),
-            "sspl_setup init --config %s" % (self.sample_global_config),
-            "sspl_setup config --config %s" % (self.sample_global_config),
-            "sspl_setup check"
+            "%s post_install --config yaml://%s" % (setup_script, self.sample_global_config),
+            "%s init --config yaml://%s" % (setup_script, self.sample_global_config),
+            "%s config --config yaml://%s" % (setup_script, self.sample_global_config)
         ]
         if self.CLUSTER_NODES:
-            # WARN: Configuring RMQ cluster will remove existing cluster setup.
-            commands.append("join_cluster --nodes %s" % (self.CLUSTER_NODES))
+            # WARN: This command will reset existing cluster setup and
+            # join user requested nodes in cluster
+            commands.append("%s join_cluster --nodes %s" % (setup_script, self.CLUSTER_NODES))
+
         # Execute setup commands
         for cmd in commands:
-            print("INFO: Executing command: 'sspl_setup %s'" % (cmd))
-            main(cmd.split())
+            print("INFO: Executing command: '%s'" % (cmd))
+            os.system(cmd)
```

Comment:
no os.system() supposed to be used, use Simple process, this is the very reason to change it to python, to reuse code.

Reply:
os.system() exposes stdout, stderr and rc in terminal for command execution.
But not SimpleProcess. Looking forward a way to do this.

Reply:
os.system() was replaced almost in whole codebase because of security concerns. We need to avoid it.

Reply:
Done